deploy/ocm-agent-operator-managedfleetnotifications/nodepool-provisioning-failure.yaml

```diff
@@ -20,5 +20,5 @@ spec:
 
       If you need assistance, please open a support case.
     resendWait: 24
-    severity: Warning
+    severity: critical
```

Comment:
We need to refer the severity mentioned in https://github.com/openshift/ops-sop/blob/master/v4/howto/onboard-an-ocmagent-alert.md#guidelines-for-updating-the-managednotification-custom-resource SOP. 
> Severity can be one of Info, Warning, Error, Fatal. Choose the most appropriate based on the situation.

Warning or Info severity is fine. That's the severity for other two MFN too.